### PR TITLE
Feat/revamp review process

### PR DESCRIPTION
**Pull request by lfsevergnini**
Sends the entire PR diff in a single request instead of multiple unrelated chunks.

The idea here is to improve the context given to the GPT model so that it can provide better comments.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -88,80 +89,74 @@ function getDiff(owner, repo, pull_number) {
...
-  
 Pull request title: ${prDetails.title}
 Pull request description:
 
 ---
 ${prDetails.description}
 ---
 
-Git diff to review:
+TAKE A DEEP BREATH AND WORK ON THIS THIS PROBLEM STEP-BY-STEP.
```
Comment (maxarias-io) [inline dist/index.js:128]: 👏

File: dist/index.js [hunk 2]
```diff
@@ -182,17 +177,18 @@ function getAIResponse(prompt) {
...
-        }
+function createComments(changedFiles, aiResponses) {
+    return aiResponses
+        .flatMap((aiResponse) => {
+        var _a;
+        const file = changedFiles.find((file) => file.to === aiResponse.file);
         return {
             body: aiResponse.reviewComment,
-            path: file.to,
+            path: (_a = file === null || file === void 0 ? void 0 : file.to) !== null && _a !== void 0 ? _a : "",
```
Comment (maxarias-io) [inline dist/index.js:187]: Is `void 0` valid?

**Review by maxarias-io — APPROVED** [review]
LGTM

File: dist/index.js (hunk 2 above)
Comment (lfsevergnini) [reply dist/index.js:187]: Nevermind these files, they're the build generated JS.

Comment (maxarias-io) [reply dist/index.js:187]: Oh duh 🤦

File: dist/index.js (hunk 1 above)
Comment (Arbust908) [inline dist/index.js:128]: NICE

**Review by Arbust908 — APPROVED** [review]
SO NICE! 👌

**Pull request merged**